package/kernel/linux/modules/other.mk

```diff
@@ -373,6 +373,23 @@ endef
 $(eval $(call KernelPackage,mmc))
 
 
+define KernelPackage/mvsdio
+  SUBMENU:=$(OTHER_MENU)
+  TITLE:=Marvell MMC/SD/SDIO host driver
+  DEPENDS:=+kmod-mmc @TARGET_kirkwood
```

Comment:
Why is this added here and not in target modules.mk? Do you expect it to be reused by other target?

Reply:
Found the comment about this as well ...